### DISCUSSION TITLE
Timestand timeshift between hunt interface and elastic

### DISCUSSION THREAD

**Discussion by ByteAndBits** [Security-Onion-Solutions/securityonion#12212] [2.4] [answered]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

64

### RAM

512GB

### Storage for /

15TB

### Storage for /nsm

30TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hi,

we have a timestamp timeshift betweeen Hunt Interface und elastic discover.

In the discover interface it´s correct in Hunt it´s +1h.

It is a custom build index template.

Where are the timeshift from?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by reyesj2** [comment, marked as answer]
https://docs.securityonion.net/en/2.4/timezones.html

In SOC under options you can set the timezone
